packages/core/package.json

```diff
@@ -206,12 +212,12 @@
   "scripts": {
     "lint": "eslint .",
     "test": "jest",
-    "build": "rm -rf ./dist && NODE_OPTIONS=\"--max-old-space-size=8192\" bunchee",
+    "build": "rm -rf ./dist && NODE_OPTIONS=\"--max-old-space-size=16384\" bunchee",
```

Comment:
Memory usage is too much now. I will try to create a manual rollup config

Reply:
8192 is not working, but 12288 is - so i keep that value for now

Reply:
bunchee is great, but it doesn't seem mature enough yet